<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SudokuWindow</class>
 <widget class="QMainWindow" name="SudokuWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sudo Caen</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="Displayer" name="displayer" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>455</width>
      <height>455</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="newGameButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>New Game</string>
    </property>
   </widget>
   <widget class="QComboBox" name="difficultyComboBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>67</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Level</string>
    </property>
   </widget>
   <widget class="QWidget" name="inputWidget" native="true">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>85</y>
      <width>450</width>
      <height>450</height>
     </rect>
    </property>
   </widget>
   <widget class="QTimeEdit" name="clockTimeEdit">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>118</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>519</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNewGame"/>
    <addaction name="actionQuit"/>
    <addaction name="actionView_Scores"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionControls"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionControls">
   <property name="text">
    <string>Controls</string>
   </property>
  </action>
  <action name="actionNewGame">
   <property name="text">
    <string>New Game</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionView_Scores">
   <property name="text">
    <string>View Scores</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Displayer</class>
   <extends>QWidget</extends>
   <header>displayer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
